src/extension/settingsHelper.ts

```diff
@@ -41,7 +41,7 @@ export class SettingsHelper {
                 return ConfigurationReader.readString(tsdk);
             }
         }
-        return null;
+        return "";
```

Comment:
Please don't change the return type of this method - there is a check in `intellisenseHelper` that checks tsdk value against `null` (https://github.com/Microsoft/vscode-react-native/blob/master/src/extension/intellisenseHelper.ts?utf8=%E2%9C%93#L199)